test/oai/test_utils.py

```diff
@@ -5,11 +5,9 @@
 import logging
 import tempfile
 from unittest import mock
+from unittest.mock import patch
 import autogen  # noqa: E402
 
-KEY_LOC = "notebook"
-OAI_CONFIG_LIST = "OAI_CONFIG_LIST"
```

Comment:
We are currently trying to improve the test utils by having a centralized way to obtain the configs. @afourney do you think we should keep these variables in test_utils.py for now?

Reply:
Specifically in `test_utils.py` those 2 file names are no longer needed, all is covered by mocks/fakes, the tests do not depend on external model configs.

Speaking of consolidating and tiding up models used by tests, I have already added `--skip-openai` flag to `pytest`: https://github.com/microsoft/autogen/pull/1097

The next step could be smth like this: https://github.com/microsoft/autogen/pull/1052#discussion_r1436008853

If there're any other ideas/issues in regards to this (besides #1089) - please share

Reply:
Got it. Thanks!